main.go

```diff
@@ -313,17 +313,16 @@ func main() {
 			if audience == "" {
 				if len(remainingArgs) > 0 {
 					audience = remainingArgs[0]
-				} else {
-					fmt.Println("Missing audience argument for JWT")
-					return
 				}
 			}
 
 			// JWT flow requires empty Scope.
 			// Also, JWT currently does not work with STS.
 			settings = &util.Settings{
+				AuthMethod:      util.MethodJWT,
```

Comment:
Let's use "AuthType: util.AuthTypeJWT" here to be consistent with the cli parameters and avoid introducing new lingo.